prisma/seed.ts

```diff
@@ -50,80 +50,6 @@ async function seed() {
     ],
   });
 
-  await prisma.record.createMany({
-    data: [
```

Comment:
Just curious as to why we are deleting all this code. Shouldn't we keep it for local testing?

Reply:
From the description, he's removing it because the seeded records are not valid, since they are only in the db and not Route53.  From this point on, we have to create the records properly in both at runtime.